cmd/single/start.go

```diff
@@ -147,6 +147,20 @@ func startPropeller(ctx context.Context, cfg Propeller) error {
 	}
 
 	if !cfg.DisableWebhook || !cfg.Disabled {
+		handlers := map[string]http.Handler{
+			"/k8smetrics": promhttp.HandlerFor(metrics.Registry, promhttp.HandlerOpts{
```

Comment:
why a separate endpoint? we want to do it so that scraping can be specialized?

Reply:
The `controller-runtime` package automatically populates a prometheus `Registry` as `metrics.Registry`. So this was just the easiest way to surface that registry (ie. separate HTTP endpoint). We could wrap both the `controller-runtime` prometheus registry and our registry (buried in flytestdlib) into a single entity, but it would require a good deal of refactoring. Also it does allowed for specialized scraping too.

Reply:
Is there any advantage to unifying the registries?